tests/ci_build/Dockerfile.gpu_build_centos7

```diff
@@ -33,9 +33,21 @@ ENV PATH=/opt/mambaforge/bin:/usr/local/ninja:$PATH
 ENV CC=/opt/rh/devtoolset-9/root/usr/bin/gcc
 ENV CXX=/opt/rh/devtoolset-9/root/usr/bin/c++
 ENV CPP=/opt/rh/devtoolset-9/root/usr/bin/cpp
+ENV CUDAHOSTCXX=/opt/rh/devtoolset-9/root/usr/bin/c++
 
 ENV GOSU_VERSION 1.10
 
+# Install RMM
+RUN git clone -b v23.04.00 https://github.com/rapidsai/rmm.git --recurse-submodules --depth 1 && \
```

Comment:
I think we are on 23.06 now https://github.com/dmlc/xgboost/blob/6e18d3a290d67943dda4d43c76f78b3c5ba4719b/tests/buildkite/conftest.sh#L27 .